client/lcd/lcd_test.go

```diff
@@ -1247,16 +1233,10 @@ func doBeginRedelegation(t *testing.T, port, seed, name, password string,
 	chainID := viper.GetString(client.FlagChainID)
 
 	jsonStr := []byte(fmt.Sprintf(`{
```

Comment:
Is this a type anywhere?

Reply:
what do you mean ?

Reply:
i mean, why are the tests written with `fmt.Sprintf`ed json blobs. Don't we have those types somewhere we can import
[](url)

Reply:
no we don't the structs specified on `stake/client/rest` use the sdk types. I think we want to test the stringified JSON here request body. Maybe we can move this to a separate `req_body_utils.go` file eventually ?